packages/shared/lib/services/file/remote.service.ts

```diff
@@ -195,83 +195,81 @@ class RemoteFileService {
 
     async zipAndSendPublicFiles({
         res,
-        integrationName,
+        scriptName,
         providerPath,
         flowType
     }: {
         res: Response;
-        integrationName: string;
+        scriptName: string;
         providerPath: string;
         flowType: string;
     }): Promise<void> {
+        // TODO: handle zero yaml here
```

Comment:
for template we don't know yet if they are zero or not. Not a big issue rn since there aren't one deployed